tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_report.py

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+from marionette_driver import expected, By, Wait
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.base import Base
+from gaiatest.apps.messages.app import Messages
+
+
+class TestSmsReport(GaiaTestCase):
+
+    def test_sms_report(self):
+        """This test sends a text message to itself. It waits for a reply message.
+           Then it checks the info in the report.
+        """
+        _text_type = (By.CSS_SELECTOR, 'p[data-l10n-id="message-type-sms"]')
+        _sender = (By.CSS_SELECTOR, 'a[data-name]')
+        _receiver = (By.CSS_SELECTOR, 'a.suggestion.js-contact-info')
+        
+        # Configuring the test
+        time_sent = time.time()
+        _text_message_content = "Automated Test %s" % str(time_sent)
+        _test_phone_number = self.testvars['phone_numbers']
+        
+        error_range_sent_in_seconds = 1200
+        error_range_rec_in_seconds = 3000
+        
+        # Sending the message
+        self.messages = Messages(self.marionette)
+        self.messages.launch()
+
+        new_message = self.messages.create_new_message(recipients=_test_phone_number,
+                                                       message=_text_message_content)
+        self.message_thread = new_message.tap_send()
+        self.message_thread.wait_for_received_messages()
+
+        last_received_message = self.message_thread.received_messages[-1]
+        last_message = self.message_thread.all_messages[-1]
+        
+        # Long Tap on received message and check the report: SMS sent received and time
+        report = last_received_message.open_report()
+
+        # Checking that the data in the received report is correct
+        # Type of message
+        self.assertTrue(report.is_message_an_sms)
+
+        # From, sender is the right one
```

Comment:
This comment doesn't add any value to the line below.

The problems with these type of comment are the following:
* Each time you change the test, you have to change the comment. The effort is duplicated.
* If somebody forgets to update the comment one day, the comment will lie
* If a reader is used to read the comments, he will end up stopping to read the code. Then bugs will be harder to find, especially if there are lying comments.
* If you get used to comment, you can loose the habit to try to create readable code. What's the point of creating straightforward code, if everybody reads only the comments? :smiley: 

In other words, let's remove this comment.